src/util/adb.js

```diff
@@ -271,14 +272,17 @@ export default class ADBUtils {
       value: `-profile ${deviceProfileDir}`,
     }];
 
-    if (!apkComponent) {
-      apkComponent = '.App';
-    } else if (!apkComponent.includes('.')) {
-      apkComponent = `.${apkComponent}`;
```

Comment:
This original logic should be kept. It allows users to use a custom app that isn't in the hardcoded list. For example if they develop their own GeckoView-based app.

The change that you need is to check if `apk` starts with any of the known prefixes, and if it is, prepend that prefix before `.${apkComponent}`.